g3/generate/client.py

```diff
@@ -25,10 +25,14 @@ def generate(self, prompt: List[Dict] | str, **kwargs: Any) -> str:
 
 
 class OpenAIChat(OpenAI):
-    def generate(self, prompt: List[Dict] | str, **kwargs: Any) -> str:
+    def stream(self, prompt: List[Dict] | str, **kwargs: Any) -> Generator:
```

Comment:
How about keeping both `generate` and `stream` methods? We might need to expose the option to the config in the future